docs/tutorials/websockets.rst

```diff
@@ -28,7 +29,43 @@ If you wish to use the advanced solution as a starting point:
 
     This tutorial can also be built on any other advanced solution, such as the Tethys Quotas tutorial solution.
 
-1. Django Channels Consumers
+1. Extending Templates
+======================
+In order to limit these notifications to the home page, we will be extending the map layout template to include a message box that will display when a new dam is added. 
+
+1. Begin by adding a new file called ``home.html`` to the ``templates/dam_inventory`` directory. Add the following code to the file:
+
+    .. code-block:: html+django
+
+        {% extends "tethys_layouts/map_layout/map_layout.html" %}
+        {% load tethys_gizmos %}
+
+
+    This code extends the default map layout template, while allowing us to make manual additions later. It also prepares the template for including 
+    custom Tethys Gizmos later.
```

Comment:
Remove the newline between these two lines since they are the same paragraph. Only use newlines between paragraphs. Use the word wrap feature of your editor to wrap the text while you are editing.